src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1552,37 +1602,57 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
     updateBaroPressure(_mode) {
         let baroPreset = SimVar.GetSimVarValue("L:XMLVAR_Baro1_SavedPressure", "number") / 16;
         
-        if (this.pressureSVG) {
+        if (this.pressureSVGLeftPart) {
             var units = Simplane.getPressureSelectedUnits();
             var pressure = Simplane.getPressureValue(units);
 
             if (_mode == "STD") {
                 if (this._baroPresetChanged !== baroPreset && this._baroPresetChanged !== 0) {
                     this.pressureBoxGroup.setAttribute("visibility", "visible");
-                    this.pressureSVG.removeAttribute("stroke", "black");
-                    this.pressureSVG.removeAttribute("stroke-width", "5px");
+                    this.pressureSVGLeftPart.removeAttribute("stroke", "black");

```

Comment:
- I'm not sure what all this attribute adding and removing is for (I know it's been there before).

- Unless some parent element in the SVG defines it otherwise, the attribute `"stroke", "black"` is redundant.

I will check both of those notes in my sim.